docs/topics/con_server-sent-events.adoc

```diff
@@ -0,0 +1,16 @@
+[id="server-sent-events_{context}"]
+= Server-sent events
+
+Server-sent events (SSE) is a reactive HTTP-based messaging protocol.
+SSE works by establishing a connection between the event source and the client, and allowing the event source, such as an application server or a messaging broker, to use this connection to push events to the client when they occur.
+The connection remains open after the updates are pushed, and can be re-used to push subsequent updates.
```

Comment:
reused
Or we can simply say: "..., and can be used to push the subsequent updates."

Reply:
I like the phrasing your suggest. Changed:)